.env.example

```
@@ -1,3 +1,5 @@
 POSTGRES_DB=
 POSTGRES_USER=
-POSTGRES_PASSWORD=
\ No newline at end of file
+POSTGRES_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)